src/stan/variational/base_family.hpp

```diff
@@ -19,6 +19,16 @@ class base_family {
    */
   virtual int dimension() const = 0;
 
+  /**
+   * Return the approximation family's parameters as a single vector
+   */
+  virtual Eigen::VectorXd return_approx_params() const = 0;
```

Comment:
Since meanfield and fullrank have different variational parameters, I though the best way to work with them is to just have the variational class return a single vector of variational parameters.

Reply:
Makes sense for sure